base/ltspace.dtx

```diff
@@ -1217,7 +1218,23 @@
 \DeclareRobustCommand{\nobreakspace}{%
    \leavevmode\nobreak\ }
 \catcode `\~=13
-\def~{\nobreakspace{}}
+%</2ekernel>
+%<latexrelease>\IncludeInRelease{2023/11/01}%
+%<latexrelease>                 {[tilde]}{Protected tilde}%
```

Comment:
I thought we documented this argument had to be a single token, but we don't document that explicitly (although do rely on a single `\string` making the whole argument safe in a csname)

Reply:
I was worried we'd simply loose a `~` here, hence not using it

Reply:
Changed